src/handlers/no_merges.rs

```diff
@@ -88,7 +88,7 @@ so these commits will need to be removed for this pull request to be merged.
 You can start a rebase with the following commands:
 ```shell-session
 $ # rebase
-$ git rebase -i master
+$ git pull --rebase https://github.com/rust-lang/rust.git master
```

Comment:
This will be wrong (and dangerously so) for other repositories, won't it?

Reply:
Damn, I completely forgot that we also use triagebot for other repos. Let me revert this.